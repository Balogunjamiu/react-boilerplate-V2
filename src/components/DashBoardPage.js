import React from 'react'


export const DashBoardPage = () =>(
    <div>
       Dashboard page content
    </div>
)
export default DashBoardPage