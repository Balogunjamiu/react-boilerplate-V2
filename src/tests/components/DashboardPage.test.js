import React from 'react'
import {shallow} from 'enzyme'
import DashBoardPage from "../../components/DashBoardPage";

test('should test if DashboardPage renders correctly', ()=>{
    const wrapper = shallow(<DashBoardPage/>)
    expect(wrapper).toMatchSnapshot()
})