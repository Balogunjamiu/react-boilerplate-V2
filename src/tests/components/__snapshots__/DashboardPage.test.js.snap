// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test if DashboardPage renders correctly 1`] = `
<div>
  Dashboard page content
</div>
`;
